### PR TITLE
docs: small documentation tweaks

### PR DESCRIPTION
**Pull request by tomleo**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Bump Public Works section to the top
- Remove some out of date documentation
- Link to [deployment confluence page](https://energysage.atlassian.net/wiki/spaces/DSE/pages/208568321/Deploy+Design+System+Documentation)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [ ] I have documented testing approach

### CODE REVIEW

File: es-design-system/README.md [hunk 1]
```diff
@@ -3,20 +3,3 @@
 ## Documentation
 
 To get started working with the design system locally see the [root README.md](../README.md)
-
-### The es-design-system Documentation
```
Comment (tomleo) [inline es-design-system/README.md:7]: Moved to https://energysage.atlassian.net/wiki/spaces/DSE/pages/208568321/Deploy+Design+System+Documentation

File: es-bs-base/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ An EnergySage branded Bootstrap
 
 ## Getting Started
 
-`npm install @energysage/es-bs-base`
+`npm install --save @energysage/es-bs-base`
```
Comment (ericdouglaspratt) [inline es-bs-base/README.md:7]: Doesn't `npm install` now save into dependencies by default, even without the `--save` param?
https://docs.npmjs.com/cli/v6/commands/npm-install

File: es-vue-base/README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ An EnergySage Vue component library
 
 ## Getting Started
 
-`npm install @energysage/es-bs-base @energysage/es-vue-base bootstrap-vue`
+`npm install --save @energysage/es-bs-base @energysage/es-vue-base bootstrap-vue`
```
Comment (ericdouglaspratt) [inline es-vue-base/README.md:7]: Same comment for this `--save` as above

**Review by ericdouglaspratt — APPROVED** [review]
Looks good overall!

File: es-bs-base/README.md (hunk 2 above)
Comment (tomleo) [reply es-bs-base/README.md]: I didn't know that was the default 👍, reverted via 0629cfd4203edacf63f88670f2bbcfb8f600d799

**Pull request merged**